### ISSUE TITLE
About 3.3.Identity Preservation Implementation

### ISSUE THREAD

**Issue by xesdiny** [Fantasy-AMAP/fantasy-talking#53]
I read the code and debugged it and found that the method mentioned in [Chapter 3.3 of the paper](https://arxiv.org/abs/2504.04842) to use arcface to intercept the face information and make Q-former feature representation to obtain the identity information has not been implemented yet.
I think it should be implemented by loading insightface similar to [MemoFaceAnalysis](https://github.com/memoavatar/memo/blob/a5aee30120d7d2fc4e0831ca61cebdbce595324b/memo/utils/vision_utils.py#L67) in [Memo](https://arxiv.org/abs/2412.04448) to obtain the identity information of dim=512, and then implemented in the forward process of `WanCrossAttentionProcessor` after face_proj conversion.

```python
    def __call__(
        self,
        attn: nn.Module,
        x: torch.Tensor,
        context: torch.Tensor,
        context_lens: torch.Tensor,
        audio_proj: torch.Tensor,
        audio_context_lens: torch.Tensor,
        latents_num_frames: int = 21,
        audio_scale: float = 1.0,
        **identiy_proj: torch.Tensor =None,**
        **identiy_context_lens: torch.Tensor = None,**
        **audio_alpha:int = 0.5,**
        **identiy_alpha: int = 0.5,**
    ) -> torch.Tensor:
        """
        x:              [B, L1, C].
        context:        [B, L2, C].
        context_lens:   [B].
        audio_proj:   [B, 21, L3, C]
        audio_context_lens: [B*21].
        """
        context_img = context[:, :257]
        context = context[:, 257:]
        b, n, d = x.size(0), attn.num_heads, attn.head_dim

        # compute query, key, value
        q = attn.norm_q(attn.q(x)).view(b, -1, n, d)
        k = attn.norm_k(attn.k(context)).view(b, -1, n, d)
        v = attn.v(context).view(b, -1, n, d)
        k_img = attn.norm_k_img(attn.k_img(context_img)).view(b, -1, n, d)
        v_img = attn.v_img(context_img).view(b, -1, n, d)
        img_x = flash_attention(q, k_img, v_img, k_lens=None)
        # compute attention
        x = flash_attention(q, k, v, k_lens=context_lens)
        x = x.flatten(2)
        img_x = img_x.flatten(2)

        if len(audio_proj.shape) == 4:
            audio_q = q.view(b * latents_num_frames, -1, n, d)  # [b, 21, l1, n, d]
            ip_key = self.k_proj(audio_proj).view(b * latents_num_frames, -1, n, d)
            ip_value = self.v_proj(audio_proj).view(b * latents_num_frames, -1, n, d)
            audio_x = attention(
                audio_q, ip_key, ip_value, k_lens=audio_context_lens
            )
            audio_x = audio_x.view(b, q.size(1), n, d)
            audio_x = audio_x.flatten(2)
        elif len(audio_proj.shape) == 3:
            ip_key = self.k_proj(audio_proj).view(b, -1, n, d)
            ip_value = self.v_proj(audio_proj).view(b, -1, n, d)
            audio_x = attention(q, ip_key, ip_value, k_lens=audio_context_lens)
            audio_x = audio_x.flatten(2)
        **if  identiy_proj:**
            # Implement identiy_x attn ouput
            # identiy_kv proj & no-mask 
             .....
           **  x = x + img_x + audio_x * audio_scale * audio_alpha + identiy_x * identiy_alpha**

        else:
            ** x = x + img_x + audio_x * audio_scale** 
        x = attn.o(x)
        return x
```
